src/components/TrustedTimestamping/FaqModal.vue

```diff
@@ -16,7 +16,7 @@
               img.no-border(
                 alt="Vue logo"
                 style="max-width: 200px; margin: 0px auto;"
-                src="../assets/logo-nn.png")
+                src="img/stellar.png")
```

Comment:
My goal was to setup these FAQs to be network agnostic which is why my goal was to use the moontography logo instead of the stellar one. Is there a particular reason you thought the stellar logo was the right one here?

Reply:
@whatl3y I guess I just saw that we were leveraging Stellar somewhere for the time stamping so thought that icon would be relevant. We can either use the moontography logo or maybe no logo?